doc/man7/RAND_DRBG.pod

```diff
@@ -264,6 +264,14 @@ from the trusted entropy sources.
 
 =back
 
+NOTE: Manual reseeding is *not allowed* in FIPS mode, because
+NIST SP-800-90A mandates that entropy *shall not* be provided by the
+consuming application, neither for instantiation, nor for reseeding.
```

Comment:
Something seems funny with the two commas.  How about:
```
consuming application: neither for instantiation nor for reseeding.
```
instead?

Reply:
I simply terminated the sentence after "application" and removed the rest, since it was dispensable.

Reply:
I'm not seeing this change yet, but that's a good fix.